ui/analyse/src/ctrl.ts

```diff
@@ -375,6 +375,7 @@ export default class AnalyseCtrl {
     }
     if (this.music) this.music.jump(this.node);
     lichess.pubsub.emit('ply', this.node.ply);
+    this.showGround();
```

Comment:
I was a bit unsure about moving this down (it needs to be after the study update) but I didn't find any reason against it and I also didn't find any issues when testing it (I tried analysis, studies, puzzles, including drawing and promotion).